### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by Daoudasultanbadolo** [blockscout/blockscout#9914] [General]
Hello how are you? please help me. I changed my phone and when I reinstalled the Openex application on the new phone, I put my 12 keywords, I found myself with a different wallet than the one I had. wallet address has changed. thank you for helping me

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
